src/Type/TermObject/Connection/TermObjectConnectionResolver.php

```diff
@@ -130,7 +130,7 @@ public static function get_query_args( $source, array $args, AppContext $context
 					$query_args['object_ids'] = $source->ID;
 					break;
 				case $source instanceof \WP_Term:
-					$query_args['object_ids'] = $GLOBALS['post']->ID;
+					$query_args['object_ids'] = $GLOBALS['post'] ? $GLOBALS['post']->ID : null;
```

Comment:
I like this change. Maybe let's make it a little more safe and make sure the global $post has an ID as well, just in cases something other than a Post was set as the global Post, which is unfortunately a possibility: 

Something like:
```
$query_args['object_ids'] = is_a( $GLOBALS['post'], 'WP_Post' ) ? $GLOBALS['post']->ID : null;
```